client/scripts/identifiers.ts

```diff
@@ -44,13 +43,13 @@ export const proposalSlugToClass = () => {
   if (app.chain.network === ChainNetwork.Kusama || app.chain.network === ChainNetwork.Polkadot) {
     mmap.set('technicalcommitteemotion', (app.chain as any).technicalCommittee);
   }
-  if (MolochTypes.EventChains.find((c) => c === app.chain.network)) {
+  if (app.chain.network === ChainNetwork.Moloch) {
     mmap.set('molochproposal', (app.chain as any).governance);
```

Comment:
Could this be replaced with `ProposalType.MolochProposal`? Same goes for the other hardcoded strings